view/range/index.ts

```diff
@@ -26,8 +28,14 @@ function initRange(
   let div = document.querySelector<HTMLDivElement>(`.range.is-${type}`)!
   let range = div.querySelector<HTMLInputElement>('.range_input')!
 
-  range.addEventListener('change', () => {
-    current.setKey(type, parseFloat(range.value))
+  range.addEventListener('input', () => {
+    let renderingValue = isRendering.get()
+
+    postponed.set({ ...current.get(), [type]: parseFloat(range.value) })
+
+    if (!renderingValue[type]) {
```

Comment:
Can we hide this logic in the store to have more high-level API in `view/`.

The idea of Nano Stores to hide as much as possible in stores.

Reply:
Yes, that is a good idea